### PR TITLE
Modify README of TizenRT and boards

### PR DESCRIPTION
**Pull request by sunghan-chang**
Some caution sentences are added.
1. Add recommandation to do TizenRT quick start first before setting board env
2. Add how to check configuation list
3. Add caution of exporting path

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: README.md [hunk 1]
```diff
@@ -38,8 +40,12 @@ Configure the build from *$TIZENRT_BASEDIR/os/tools* directory
...
 ./configure.sh <board>/<configuration_set>
 ```
-For list of boards and configuration set supported, refer belows.  
-Above copies the canned configuration-set for the particular board, into the *$TIZENRT_BASEDIR/os* directory.  
+The folder list under *build/configs* are \<boards\> and each \<boards\> folder contains \<configuation_set\> folder.  
+Easy way to check the list of \<boards\> and \<configuation_set\> is typing wrong argument like below:
+```bash
+./configue.sh a
+```
+
```
Comment (dr-venkman) [inline README.md]: ./configure.sh --help gets you the list of board and configurations. It is a better choice to present than entering a wrong combination.

File: build/configs/artik053/README.md [hunk 2]
```diff
@@ -116,11 +119,9 @@ SUBSYSTEMS=="usb",ATTRS{idVendor}=="0403",ATTRS{idProduct}=="6010",MODE="0666" R
 
 ## How to program a binary
 
-There are two methods, using OpenOCD or script.
-
-After building TizenRT, follow below steps at $TIZENRT_BASEDIR/os folder.
-
-TIZENRT_BASEDIR was set at [[Getting the sources]](../../../README.md#getting-the-sources) tab of Quick Start.
+There are two methods, using OpenOCD or script.
```
Comment (dr-venkman) [inline build/configs/artik053/README.md]: would be nice to distinguish between the two methods. As in, to tell when one method would be preferable over another.

File: build/configs/artik053/README.md [hunk 3]
```diff
@@ -116,11 +119,9 @@ SUBSYSTEMS=="usb",ATTRS{idVendor}=="0403",ATTRS{idProduct}=="6010",MODE="0666" R
...
 ## How to program a binary
 
-There are two methods, using OpenOCD or script.
-
-After building TizenRT, follow below steps at $TIZENRT_BASEDIR/os folder.
-
-TIZENRT_BASEDIR was set at [[Getting the sources]](../../../README.md#getting-the-sources) tab of Quick Start.
+There are two methods, using OpenOCD or script.  
+After building TizenRT, follow below steps at $TIZENRT_BASEDIR/os folder.  
+TIZENRT_BASEDIR was set at [[Getting the sources]](https://github.com/Samsung/TizenRT#getting-the-sources) tab of Quick Start.
```
Comment (dr-venkman) [inline build/configs/artik053/README.md]: The last sentence can be replaced by a link in the previous sentence itself.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**